templates/corporate/history.md

```diff
@@ -129,6 +129,16 @@ badge](https://www.capterra.com/p/197945/Zulip/).
 
 ## Press highlights
 
+- June 2024: An in-depth [review of
+  Zulip](https://www.hostingadvice.com/blog/emerging-open-source-team-chat-app-set-to-rival-slack/)
+  is published on [HostingAdvice.com](https://www.hostingadvice.com/).
+
+> "An excellent solution for teams collaborating across different time zones."
+>
+> — [Zulip
+> review](https://www.hostingadvice.com/blog/emerging-open-source-team-chat-app-set-to-rival-slack/)
+> on *HostingAdvice.com*
```

Comment:
This is fine, but using [markdown references](https://www.markdownguide.org/basic-syntax/#reference-style-links) makes the source a lot readable when using a long link, and I much prefer when those are used :).